### DISCUSSION TITLE
strimzi schema registry

### DISCUSSION THREAD

**Discussion by vyom-soft** [strimzi/strimzi-kafka-operator#8915] [General]
Hello,
I would like to know how does strimzi-kafka support schema registry.
Which schema registry can be integrated with strimzi-kafka.

Thank you.

**Comment by ppatierno** [comment]
Hi,
Strimzi doesn't have any officially supported schema registry. In general it's a matter of deploying it and connecting to Kafka but there is no way to "integrate" within the Strimzi operator (I mean, having the operator deploying a specific schema registry).
We saw different users in the community using their own schema registry with no issues (i.e. the Confluent one).
